src/accounts/accounts.service.ts

```diff
@@ -21,6 +21,8 @@ interface IAccountsResponse {
 
 @Injectable()
 export class AccountsService {
+  private readonly logger = new Logger('AccountsService');
```

Comment:
This we can do like this `private readonly logger = new Logger(AccountsService.name);`